packages/handlersjs-http/lib/servers/node/node-http-request-response.handler.spec.ts

```diff
@@ -315,4 +354,32 @@ describe('NodeHttpRequestResponseHandler', () => {
 
   });
 
+  describe('parseBody', () => {
```

Comment:
What if any of the parsers fail?

Reply:
There doesn't seem to be a catch implemented in the handler for that. Should I add a try catch block in the function?

Reply:
No, we someday should, but for now we should write a test saying that it errors when the syntax is wrong.